### ISSUE TITLE
No setting available to create auto-backup in VPS management section

### ISSUE THREAD

**Issue by cavitnation** [vultr/whmcs-vultr#78]
Hi,

There is No setting available to create auto-backup in VPS management section. In configuration option we can set the price of auto-backup. As we know that vultr auto-backup price is %20 of the VPS hourly/monthly cost. 

Once VPS is created, there is no setting to create/setup autobackups by different period of time.

Link: https://ibb.co/d5SXWVR

**Comment by SaneChoice** [comment]
I agree and cannot see a way to configure auto backup timings.  (See my screenshot). I think this module only allows you to SEE Backups, not configure.

Could a developer confirm the current behaviour of Auto Backups in the module? 

<img width="857" alt="Screenshot 2020-04-27 at 12 37 46" src="https://user-images.githubusercontent.com/64412037/80368110-1ce43580-8884-11ea-9c8e-3b32f99074ab.png">
